unity-renderer/Assets/DCLServices/Lambdas/LambdasService.cs

```diff
@@ -67,15 +67,20 @@ public class LambdasService : ILambdasService
             params (string paramName, string paramValue)[] urlEncodedParams)
         {
             string urlWithParams = AppendQueryParamsToUrl(url, urlEncodedParams);
-            UnityWebRequest request = await webRequestController.Ref.GetAsync(urlWithParams, requestAttemps: attemptsNumber, timeout: timeout, isSigned: isSigned, signUrl: signUrl, cancellationToken: cancellationToken);
-
-            if (request.result != UnityWebRequest.Result.Success)
-                return (default(TResponse), false);
-
-            string text = request.downloadHandler.text;
 
             try
             {
+                UnityWebRequest request = await webRequestController.Ref.GetAsync(urlWithParams,
```

Comment:
This method already handles returning a `(default(TResponse), false)` in the case something goes wrong, I believe that a 404 should also return a false instead of an exception

Reply:
never mind, found a better way to handle this